src/psyclone/nemo.py

```diff
@@ -569,3 +552,16 @@ def match(node):
                 if colons:
                     return False
         return True
+
+    def reference_accesses(self, var_accesses):
+        '''Get all variable access information. The loop variable is
```

Comment:
Incorrect docstring - this method doesn't do anything other than call `var_accesses.next_location`. Should the TODO refer to #440?

Reply:
I updated the docstring and changed the TODO to #440